Teaser block docs (#4461)
plone/volto: docs/source/configuration/component-registry.md

---
myst:
  html_meta:

>     "description": "Volto provides a component registry integrated that store named references to components that can be queried programatically"

Reviewer: ```suggestion
    "description": "Volto provides an integrated component registry that stores named references to components, allowing them to be queried programmatically."
```

>     "property=og:description": "Volto provides a component registry integrated that store named references to components that can be queried programatically"

Reviewer: ```suggestion
    "property=og:description": "Volto provides an integrated component registry that stores named references to components, allowing them to be queried programmatically."
```

>     "property=og:title": "Component Registry"

Reviewer: ```suggestion
    "property=og:title": "Component registry in Volto"
```

    "keywords": "Volto, Plone, frontend, React, registry, component"
---

(component-registry)=

> # Component Registry

Reviewer: Sentence case for headings.

```suggestion
# Component registry
```
